TransIT.BLL/DTOs/PartInDTO.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace TransIT.BLL.DTOs
+{
+    public class PartInDTO
+    {
+        private uint amount;
+        private float price;
+
+        public PartInDTO()
+        {
+
+        }
+
+        public int Id { get; set; }
+        public int UnitId { get; set; }
+        public int? PartId { get; set; }
+        public int CurrencyId { get; set; }
+
+        public string Batch { get; set; }
+        public uint Amount
+        {
```

Comment:
use attributes for such purposes, like [MinLength(0)] or write your own if you need to throw an exception, and it can be re-used in the future